.github/workflows/setup-cluster-env/action.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+name: setup-cluster-env
+description: Setup cluster env
+
+inputs:
+  vault-url:
+    description: 'Vault URL'
+    required: false
+  vault-role-id:
+    description: 'Vault role ID'
+    required: false
+  vault-secret-id:
+    description: 'Vault secret ID'
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-go@v3
+      with:
+        go-version-file: go.mod
+        cache: true
+        cache-dependency-path: |
+          go.sum
+          tools/go.sum
+    - uses: hashicorp/vault-action@v2.5.0
+      with:
+        url: ${{ inputs.vault-url }}
+        roleId: ${{ inputs.vault-role-id }}
+        secretId: ${{ inputs.vault-secret-id }}
+        method: approle
+        secrets: |
+          ${{ env.EC_KEY_SECRET }} apiKey | EC_API_KEY ;
+          ${{ env.AWS_ACCOUNT_SECRET }} user | AWS_PROFILE ;
+          ${{ env.AWS_ACCOUNT_SECRET }} csv | AWS_ACCOUNT_IMPORT
```

Comment:
These env variables are defined in the consumers of this reusable action, since they are the same, will it make sense to set those env variables within the context of this step?